.github/workflows/build.yml

```diff
@@ -74,6 +74,11 @@ jobs:
         with:
           path: ~/.linuxkit/cache
           key: linuxkit-amd64-${{ github.sha }}
+      - name: Fail if cache miss
+        uses: actions/github-script@v6
+        if: steps.cache.outputs.cache-hit != 'true'
```

Comment:
Error in CI indicates that we did not set id field for the cache step.

Reply:
Yeah, fixed.